releases/release-7.1.0.md

```diff
@@ -23,35 +23,65 @@ In v7.1.0, the key new features and improvements are as follows:
+    <td>TiFlash supports intermediate result spill to disk to mitigate OOMs in data-intensive operations such as aggregations, sorts, and hash joins.</td>
+  </tr>
   <tr>
-    <td rowspan="2">SQL</td>
+    <td rowspan="3">SQL</td>
     <td><a href="https://docs.pingcap.com/tidb/dev/sql-statement-create-index#multi-valued-index" target="_blank">Multi-valued index</a> (GA)</td>
     <td>Support MySQL-compatible multi-valued indexes and enhance the JSON type to improve compatibility with MySQL 8.0. This feature improves the efficiency of membership checks on multi-valued columns.</td>
   </tr>
+  <tr>
+    <td><a href="https://docs.pingcap.com/tidb/dev/time-to-live" target="_blank">Row-level TTL</a> (GA)</td>
+    <td>Support managing database size and improve performance by automatically expiring data of a certain age.</td>
+  </tr>
   <tr>
     <td><a href="https://docs.pingcap.com/tidb/dev/generated-columns" target="_blank">Generated columns</a> (GA)</td>
     <td>Values in a generated column are calculated by a SQL expression in the column definition in real time. This feature pushes some application logic to the database level, thus improving query efficiency.</td>
   </tr>
+  <tr>
+  <td rowspan="1">Security</td>
+    <td>Added LDAP integration (GA)</td>
+    <td>TiDB adds support for LDAP authentication. This is compatible with the <a href="https://dev.mysql.com/doc/refman/8.0/en/ldap-pluggable-authentication.html" target="_blank">MySQL implementation</a>.</td>
+  </tr>
 </tbody>
 </table>
 
 ## Feature details
 
 ### Performance
 
+* Enhance the Partitioned Raft KV storage engine (experimental) [#11515](https://github.com/tikv/tikv/issues/11515) [#12842](https://github.com/tikv/tikv/issues/12842) @[busyjay](https://github.com/busyjay) @[tonyxuqqi](https://github.com/tonyxuqqi) @[tabokie](https://github.com/tabokie) @[bufferflies](https://github.com/bufferflies) @[5kbpers](https://github.com/5kbpers) @[SpadeA-Tang](https://github.com/SpadeA-Tang) @[nolouch](https://github.com/nolouch) **tw:Oreoxmt**
+
+    TiDB v6.6.0 introduces the Partitioned Raft KV storage engine as an experimental feature, which uses multiple RocksDB instances to store TiKV Region data, and the data of each Region is independently stored in a separate RocksDB instance. The new storage engine can better control the number and level of files in the RocksDB instance, achieve physical isolation of data operations between Regions, and support stably managing more data. Compared with the original TiKV storage engine, using the Partitioned Raft KV storage engine can achieve about twice the write throughput, three times the read throughput, and reduce the elastic scaling time by about 4/5 under the same hardware conditions and mixed read and write scenarios.
+
+    In TiDB v7.1.0, the Partitioned Raft KV storage engine is compatible with TiFlash and supports tools such as TiDB Lightning, BR, and TiCDC.
```

Comment:
@tonyxuqqi  for review. I was under the impression the ecosystem support wasn't ready yet.

Reply:
@tonyxuqqi @benmaoer PTAL

Reply:
> @tonyxuqqi for review. I was under the impression the ecosystem support wasn't ready yet.

BR, TiCDC, Lightning works, but TiFlash is not ready.